gotocompany/shield/v1beta1/admin.proto

```diff
@@ -0,0 +1,1159 @@
+syntax = "proto3";
+
+package gotocompany.shield.v1beta1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "validate/validate.proto";
+
+option go_package = "github.com/goto/proton/shield/v1;shieldv1beta1";
+option java_outer_classname = "Admin";
+option java_package = "com.gotocompany.proton.shield.v1beta1";
+// These annotations are used when generating the OpenAPI file.
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "Admin",
+    version: "0.1.0";
+  };
+  schemes: HTTP;
+};
+
+service AdminService {
```

Comment:
If we update this, would there be a problem with the clients?

Reply:
We will need to update the client for admin from shieldservice to adminservice